### ISSUE TITLE
PosTag does not sort xticklabels in frequency mode

### ISSUE THREAD

**Issue by naresh-bachwani** [DistrictDataLabs/yellowbrick#856]
**Describe the bug**
I noticed a strange behavior while working with PosTag visualizer. When in frequency mode it does sort the bars but the x tick labels remain in the initial order. 
![Screenshot (17)](https://user-images.githubusercontent.com/43993586/58130671-3e7f7a80-7c3a-11e9-977f-1209dd4d9ca1.png)

**To Reproduce**

```python
corpus = load_corpus('data/hobbies')
docs = corpus.data
labels = corpus.target
tagged_stanzas = [nltk.pos_tag(nltk.word_tokenize(sent)) for sent in docs]
tag = [tagged_stanzas]

_, (ax1,ax2) = plt.subplots(1,2)

viz = PosTagVisualizer(ax=ax1)
viz.fit(tag)
viz.poof()
viz.ax.grid(False)

oz = PosTagVisualizer(frequency=True, ax=ax2)
oz.fit(tag)
oz.poof()
oz.ax.grid(False)
```
where load_corpus is the function from yellowbrick contributing [section](https://www.scikit-yb.org/en/latest/api/text/corpus.html)

**Comment by bbengfort** [comment]
This will likely be fixed in #847 - thanks @naresh-bachwani!

**Comment by bbengfort** [comment]
@naresh-bachwani we fixed this in #847 right? When we did that fix did we add a test to make sure the tick labels were sorted correctly?

**Comment by naresh-bachwani** [comment]
Yes it was fixed but we did not add tests for it. I will fix it with my new PR that adds helper function for stack bar charts.

**Comment by bbengfort** [comment]
Perfect - thanks! You can use the same test for the tick labels that you used in #870!

**Comment by bbengfort** [comment]
@naresh-bachwani Since you added the tests in #872 I'm going to go ahead and close this issue!

Fixed by #847 and #870.
